python/lsst/obs/comCam/ingest.py

```diff
@@ -18,14 +18,30 @@ def __init__(self, config, *args, **kwargs):
         super(ParseTask, self).__init__(config, *args, **kwargs)
 
     def getInfo(self, filename):
-        # Grab the basename
+        """ Get the basename and other data which is only available from the filename/path.
+        
+        This seems fragile, but this is how the teststand data will *always* be written out, 
+        as the software has been "frozen" as they are now in production mode.
+
+        Parameters
+        ----------
+        filename : `str`
+            The filename
+
+        Returns
+        -------
+        phuInfo : `dict`
+            Dictionary containing the header keys defined in the ingest config from the primary HDU
+        infoList : `list`
+            A list of dictionaries containing the phuInfo(s)
```

Comment:
I'm confused how this is different from the above (and also, `phuInfo` name again...).